src/main/webapp/web.xml

```diff
@@ -105,4 +105,31 @@
         <servlet-name>deleteManufacturer</servlet-name>
         <url-pattern>/manufacturers/delete</url-pattern>
     </servlet-mapping>
+    
+    <servlet>
+        <servlet-name>loginController</servlet-name>
+        <servlet-class>mate.controller.LoginController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>loginController</servlet-name>
+        <url-pattern>/login</url-pattern>
+    </servlet-mapping>
+    
+    <servlet>
+        <servlet-name>getMyCurrentCarsController</servlet-name>
+        <servlet-class>mate.controller.driver.GetMyCurrentCarsController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>getMyCurrentCarsController</servlet-name>
+        <url-pattern>/cars/all</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/cars/current</url-pattern>
```

`/cars/all` looks like we want to get all cars, not bu driver

Reply:
done